src/lib/functions/server.js

```diff
@@ -160,14 +206,15 @@ const createHandler = function ({ config, functionsRegistry }) {
   }
 }
 
-const getFunctionsServer = function ({ buildersPrefix, config, functionsPrefix, functionsRegistry, siteUrl }) {
+const getFunctionsServer = async function (options) {
+  const { buildersPrefix = '', functionsPrefix = '', functionsRegistry, siteUrl } = options
   // performance optimization, load express on demand
   // eslint-disable-next-line node/global-require
   const express = require('express')
   // eslint-disable-next-line node/global-require
   const expressLogging = require('express-logging')
   const app = express()
-  const functionHandler = createHandler({ config, functionsRegistry })
+  const functionHandler = await createHandler(options)
```

Comment:
`createHandler` is not async so this `await` seems redundant, unless we should be awaiting `startPollingForAPIAuthentication` here https://github.com/netlify/cli/pull/4235/files#diff-de106b743be68ae2e9146d09931f894e258783a68db916a6d0e39e3908b28c84R90

Reply:
This might be an artifact from a previous version of the code. Let me see if it can be safely removd.